src/components/app/matching/Match.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+
+export interface MatchProps {
+  matchName: string,
+  matchPicture: string,
+  matchIsStarred: boolean,
+  chatStatusIcon: string,
+}
+
+export const Match = (props: MatchProps) => {
+  return (
```

Comment:
can be simplified to only the parentheses, no braces and return statement required

Reply:
alrigth